.gitlab-ci.yml

```diff
@@ -0,0 +1,16 @@
+# See https://docs.gitlab.com/ce/ci/yaml/README.html for the documentation
+
+package:
+  image: debian:10
```

Comment:
Consider just using `python:3.9`. It's buster with python and pip preinstalled. I think even `python:3.9-alpine` could work as `build-documentation.sh` script is POSIX sh, we don't need any other utilities.

Reply:
Note that if you go that way you probably need to override entrypoint as by default python images start the REPL.
```yaml
image:
  name: python:9.3-alpine
  entrypoint: /bin/sh
```

Reply:
I would prefer to keep using plain debian. In that way we know all the dependencies.